lib/ansible/modules/identity/keycloak/keycloak_client.py

```diff
@@ -715,7 +716,16 @@ def main():
     result = dict(changed=False, msg='', diff={}, proposed={}, existing={}, end_state={})
 
     # Obtain access token, initialize API
-    kc = KeycloakAPI(module)
+    connection_header = get_token(
```

Comment:
I am missing the KeycloakErro catch in conjunction with a fail_json() here

Reply:
Done in 809cc113c51af487af91b140d6a63391e8acc2a4.